### ISSUE TITLE
Not able to build the project after updating version 3.7.0 to 7.0.1

### ISSUE THREAD

**Issue by kdshukla** [fritzlabs/fritz-ai-ios-sdk#3]
We have a objective-C project which is using following pods
pod 'Fritz', '~> 3.7.0'
pod 'Fritz/Vision'
pod 'Fritz/VisionPoseModel'
pod 'Fritz/VisionStyleModel/Paintings'

Now, I am updating these to 7.0.1 as mentioned below

pod "Fritz", "7.0.1"
pod "Fritz/VisionPoseModel/Human/Fast", "7.0.1"
pod "Fritz/VisionPoseModel", "7.0.1"
pod 'Fritz/VisionStyleModel/Paintings', "7.0.1"

But when building the App, getting errors. could not build module

<img width="1135" alt="Screenshot 2023-05-01 at 9 16 14 PM" src="https://user-images.githubusercontent.com/121655683/235480967-c5380f9d-800b-43bb-a762-46f61ed36cd1.png">
<img width="1135" alt="Screenshot 2023-05-01 at 9 15 22 PM" src="https://user-images.githubusercontent.com/121655683/235480988-6fd981ec-fbfd-4a8c-a7a6-7b847a3b15bb.png">

Can someone please guide me what's wrong with it.

**Comment by kdshukla** [comment]
@mikchmie, can you please guide me on this?

**Comment by mikchmie** [comment]
Not really. I'm not an author of this library - I merely submitted a small improvement for its Cocoapods spec. I don't know what's the problem in your case.
